src/index.ts

```diff
@@ -85,7 +85,7 @@ export default function supernova(env: Galaxy) {
       const embed = useEmbed();
 
       const [pageInfo, setPageInfo] = useState(initialPageInfo);
-      const shouldRenderVirtualizedTable = false; // layout.scrollMode === 1;
+      const shouldRenderVirtualizedTable = areBasicFeaturesEnabled && layout.presentation?.usePagination === false;
```

Comment:
use a `!` here? or are we missing a type on usePagination?

Reply:
Both `presentation` and `presentation.usePagination` are optional. But the default value is to use pagination. So need to check that `usePagination` is explicitly set otherwise it would break the default value.

Once we do change default value to we can do `!`.

Reply:
ah ok makes sense